### ISSUE TITLE
dataset download

### ISSUE THREAD

**Issue by zcyHHU** [rising-turtle/VCU_RVI_Benchmark#15]
Hi, thanks for your nice work about the RGB-D-Inertial dataset and the DUI_VIO, but I have met with a very strange problem. I want to download the dataset to test the Visual-Inertial mode on the ORB-SLAM3, but I can not download it. So, I really need some help from you about the data download. Thank you very much.I am looking forward to hearing from you. my email:zcynj@hhu.edu.cn

**Comment by lvj5077** [comment]
try this link 
https://drive.google.com/drive/folders/1yO10ty1IbC9aInYz7wdauiFoKlnjZCsc?usp=sharing

**Comment by zcyHHU** [comment]
thank you

**Comment by zcyHHU** [comment]
I would like to test the data in TUM format. Can you provide it? Or is there a script that can convert ROSbag to TUM format

**Comment by rising-turtle** [comment]
Yes, you can use the script process_result.py to make the conversion.
